apistar/client/transports.py

```diff
@@ -50,7 +54,7 @@ def send(self, method, url, query_params=None, content=None, encoding=None):
         response = self.session.request(method, url, **options)
         result = self.decode_response_content(response)
 
-        if response.status_code >= 400 and response.status_code <= 599:
+        if 400 <= response.status_code <= 599:
```

Comment:
Does this work as expected? We don’t end up with a bool <= int comparison?

Reply:
seems to work as expected
![image](https://user-images.githubusercontent.com/2430381/38654567-fc9273d4-3ddd-11e8-97a4-e4c1c97257cc.png)

it is funny, you never see this form in python usually, but it is the more mathematical way to represent such relationships